### DISCUSSION TITLE
Experience with load balanced cluster setup

### DISCUSSION THREAD

**Discussion by so010** [edumeet/edumeet#737] [General]
Please share your experience with load balanced cluster setup.

- What is your HAProxy config or did you use a different approach?
- Is the documentation here https://github.com/edumeet/edumeet/blob/master/HAproxy.md good enough or did you made some improvements.
- What are your metrics (#participants, #rooms, #machines, specs of machines, load, ?)

**Comment by zmartinovic** [comment]
We have 3 servers (8 VCPUs and 8 GB RAM) using HAproxy. We have LTI auth for our Moodle and OIDC for login via our national auth (AAI@EduHr). Our HAProxy config is similar as in the documentation:
```
balance url_param roomId
hash-type consistent
server meet01.srce.hr XXX.XXX.XXX.XXX:80 check maxconn 600 verify none
```
We are just starting to use eduMeet more intensively but auth and everything else seems to be working perfectly :) Will return with info after a couple of days.

**Reply by Alex-YddeR** [reply]
How do we configure app. do we need specify in app/config as  serverHostname : "Ha Proxy URL"?

**Reply by zmartinovic** [reply]
No, you configure it as in documentation https://github.com/edumeet/edumeet/blob/master/HAproxy.md. In server config you configure Redis, listening port and in webRtcTransport.listenIps.ip we use IP of each server. It works great for us.

**Reply by Alex-YddeR** [reply]
In our case both server and app component runs in different host.  we hosted app component on AWS CloudFront and server component on EC2 instances.

How can we configure app in this scenario.

Thanks
